gui/packages/ubuntupro/lib/pages/startup/startup_page.dart

```diff
@@ -106,6 +106,8 @@ extension AgentStateL10n on AgentState {
       case AgentState.ok:
         // This state should not need translations.
         return '';
+      default:
+        throw UnimplementedError(toString());
```

Comment:
Inacessible by design, only being caught if the enum is changed without this function being updated.